### ISSUE TITLE
Add a visualization of the codes hierarchy, from where you can rename codes 

### ISSUE THREAD

**Issue by albertocottica** [edgeryders/openethnographer#59]
One for @guywiz ?

**Comment by guywiz** [comment]
We need to discuss what "renaming" codes mean. Actually, we (UBx) need to better understand how the hierarchy relations are stored. Looking at the taxonomy term page, it seems there are different ad hoc ways that were used, colon for some (as in "solution: education"), period (as is "df.security, or a mix "df.solution: 5G"). Explicit child-parent relations also appear to be used (as with spotTheFuture/actors/communities

So it seems we'll have to manually deal with different cases and try to rebuild the hierarchy from these "orthographic" variations (and others such as singular/plural forms, etc.) :-(

Is there a view allowing to easily extract tags, and data linking them to posts/comments they tag?

**Comment by albertocottica** [comment]
@guywiz no, no. We ignore previous instances. This will be a well-formed hierarchy that reuses the taxonomy term entity in Drupal. 
"Renaming" a code means the following. Codes are taxonomy terms. They have unique identifiers (tid or Term ID); and text labels. Example:
```json
node: {
name: "3D printing",
Description: "Annotation classifications added by Open Ethnographer users",
parentName: "",
Term ID: "2,013",
Parent Term ID: "0"
}
```


Renaming means changing the label. When you do that, nothing else changes: the code's children and parent terms, the annotations etc.

**Comment by guywiz** [comment]
Looks like these terms can be obtained as a view? Am I right? Is the view already defined and usable?

**Comment by albertocottica** [comment]
@guywiz of course, and since July 2016. See this post: https://edgeryders.eu/en/open-ethnographer/fetching-openethnographer-codes-and-annotations-in
